src/web/mjs/connectors/ScanManga.mjs

```diff
@@ -7,11 +7,9 @@ export default class ScanManga extends Connector {
         super();
         super.id = 'scanmanga';
         super.label = 'ScanManga';
-        this.tags = [ 'manga', 'french', 'novel' ];
+        this.tags = ['manga', 'french', 'novel'];
         this.url = 'https://www.scan-manga.com';
-        // If not set, defaults to "fr" for french machines, which will return empty responses
-        // Only fail for "fr" language, and works for any other language, including "fr-FR" and random strings
-        this.requestOptions.headers.set('accept-language', 'en');
+        var _0x1087 = ['381sfGWON', '916955zZjSQL', '31ElJoip', '708qNyAYT', '2ayDZDy', 'accept-language', 'set', '2RuRNTr', '681974vvHWEX', '71297ucZdkv', '19023wnFPih', 'headers', '1EktpzL', 'requestOptions', '2315QJlMKR', '765166pfLgfQ', 'en-US', '2469XKKoaj']; var _0x3907 = function (_0x376230, _0x20d038) { _0x376230 = _0x376230 - 0x1be; var _0x1087ed = _0x1087[_0x376230]; return _0x1087ed; }; var _0x59f72d = _0x3907; (function (_0x6ca478, _0x265693) { var _0x302037 = _0x3907; while (!![]) { try { var _0x5c9580 = parseInt(_0x302037(0x1bf)) * -parseInt(_0x302037(0x1c4)) + parseInt(_0x302037(0x1c3)) * parseInt(_0x302037(0x1c7)) + parseInt(_0x302037(0x1ce)) + -parseInt(_0x302037(0x1cf)) * -parseInt(_0x302037(0x1c5)) + -parseInt(_0x302037(0x1ca)) * -parseInt(_0x302037(0x1c2)) + parseInt(_0x302037(0x1c9)) * -parseInt(_0x302037(0x1be)) + -parseInt(_0x302037(0x1cd)) * parseInt(_0x302037(0x1cc)); if (_0x5c9580 === _0x265693) break; else _0x6ca478['push'](_0x6ca478['shift']()); } catch (_0x5bf218) { _0x6ca478['push'](_0x6ca478['shift']()); } } }(_0x1087, 0xf353f), this[_0x59f72d(0x1c8)][_0x59f72d(0x1c6)][_0x59f72d(0x1c1)](_0x59f72d(0x1c0), _0x59f72d(0x1cb)));
```

Comment:
This obfuscation is quite useless, as it can be unveiled within a few minutes

Reply:
May just set header to same value as one of the popular browsers, so it cannot be distinguished if request is from HakuNeko or Firefox, e.g.
`this.requestOptions.headers.set('accept-language', 'en-US,en;q=0.9,de;q=0.8');`

Reply:
i mean they previously shut out all there french users to try get around HN